### PR TITLE
Palo Alto NGFW: field enhancement

### PR DESCRIPTION
**Pull request by LenaigKaliou**
## Description

Related to [this issue](https://github.com/SekoiaLab/integration/issues/1622)

## Type of Change

- [ ] New intake format
- [x] Update to existing intake format
- [ ] Bug fix
- [ ] Documentation update
- [ ] Other (please describe):

## Affected Intake Formats

- Vendor/product:

### 🔒 CRITICAL - Security Requirements (MUST BE VERIFIED FIRST)

- [ ] **No real email addresses** in test files (use: user@example.com, test@example.org)
- [ ] **No real passwords or API keys** (use: "P@ssw0rd!", "xxxxxxxxxxxx", "FAKE_TOKEN_123")
- [ ] **No real IP addresses** (use TEST-NET ranges: 192.0.2.x, 198.51.100.x, 203.0.113.x)
- [ ] **No real phone numbers** (use: +1-555-0100 or similar test numbers)
- [ ] **No real names or PII** (use: "John Doe", "Jane Smith", "User123")
- [ ] **No real company/organization names** in test data (unless public vendors)
- [ ] **No real usernames, hostnames, or domain names** (use: user1, host.example.com, test.corp)
- [ ] All test data has been anonymized and verified

### Required for All PRs

- [ ] Code has been linted
    - [ ] with the linter for manifest, smart-descriptions, and test files (`uv run python linter.py check --changes` in utils/)
    - [ ] with Prettier for the parser code files (`npx prettier --write <module>/<format>/ingest/parser.yml`)
- [ ] All CI/CD checks pass
- [ ] Changes follow the contribution guidelines in `CONTRIBUTING.md`

### Required for New/Updated Intake Formats

- [ ] Clear descriptions provided for modules and formats
- [ ] Logo files included for new modules/formats
- [ ] Parser test coverage is at least 75%
- [ ] At least one smart-description is provided
- [ ] README.md updated (if applicable)
- [ ] Taxonomy mappings are correct and documented
- [ ] Test cases cover edge cases and error handling

### Testing

- [ ] Local tests pass (`uv run pytest` in utils/)
- [ ] Sample data is representative and **completely anonymized**
- [ ] **No sensitive information** (PII, credentials, real IPs, real emails) in test files
- [ ] Test data uses example.com/example.org domains and TEST-NET IP ranges
- [ ] Parser handles malformed data gracefully

### Documentation

- [ ] Code changes are documented
- [ ] Smart-descriptions are clear and informative
- [ ] Field mappings are explained

## Related Issues

Closes #

## Summary by Sourcery

Enhance Palo Alto NGFW parsing to support additional GlobalProtect fields and align tests with the updated field mappings.

New Features:
- Map new GlobalProtect event status fields to the generic action outcome field.
- Extract connection error details into the standard error.message field.
- Populate action name from an additional GlobalProtect event identifier field.

Tests:
- Update/rename the GlobalProtect test fixture to cover the newly mapped fields.

### CODE REVIEW

File: Palo Alto Networks/paloalto-ngfw/ingest/parser.yml [hunk 1]
```diff
@@ -832,8 +832,8 @@ stages:
           action.outcome: "success"
         filter: "{{parsed_event.message.get('Action') != None}}"
       - set:
-          action.outcome: "{{parsed_event.message.Status or parsed_event.message.PanOSEventStatus}}"
-        filter: "{{parsed_event.message.Status != None or parsed_event.message.PanOSEventStatus != None}}"
+          action.outcome: "{{parsed_event.message.Status or parsed_event.message.PanOSEventStatus or parsed_event.message.EventStatus}}"
```
Comment (mchupeau-sk) [inline Palo Alto Networks/paloalto-ngfw/ingest/parser.yml]: A quick question, are we sure that parsed_event.message.EventStatus can only have : success, failure or unknown values ?

**Review by mchupeau-sk — COMMENTED** [review]
Thanks for the work ! just one question and we are good !

File: Palo Alto Networks/paloalto-ngfw/ingest/parser.yml (hunk 1 above)
Comment (LenaigKaliou) [reply Palo Alto Networks/paloalto-ngfw/ingest/parser.yml]: I think so according to the tests, but we'll need to verify or add a failsafe (for EventStatus as well as Status or PanOSEventStatus). I didn't include it in this PR as for me it was more related to [this improvement](https://github.com/SekoiaLab/integration/issues/1612), but I can add this to the filter and validate the format in the improvement ticket in the same PR if you prefer :)

**Review by mchupeau-sk — APPROVED** [review]
LGFM
